### ISSUE TITLE
Cannot publish extensions while an added owner of publisher

### ISSUE THREAD

**Issue by cloudRoutine** [Microsoft/vscode-vsce#116]
I can login as the publisher, but when I try to publish it fails. I can access the publisher management page, although it doesn't come up in my dropdown menu to switch between publishers. I was added at the owner level so it seems like something is broken here.

![](http://i.imgur.com/qvkTRQ3.png)

I also tried uploading from the management page but that failed too

![](http://i.imgur.com/7Wz2mgc.png)

**Comment by joaomoreno** [comment]
@cloudRoutine Ping on email.

**Comment by cloudRoutine** [comment]
Sorry I haven't gotten a chance to publish a release yet. The other
maintainer forgot to wait and let me try again and did the latest one
earlier today. I'm waiting on a PR to come in a couple of hours, I'll give
it a shot after I've merged that.

**Comment by joaomoreno** [comment]
ping
